serverless/functions/reference-content/code-examples.mdx

```diff
@@ -102,7 +102,7 @@ def handle(event, context):
 
 With event objects you can pass information through your HTTP request. They are composed of the following objects:
 
-* `pathParameters`: map(string)string - Parameters defined in the path of the HTTP Request
+* `path`: string - Parameters defined in the path of the HTTP Request
 * `queryStringParameters`: map(string)string - Query Strings parameters of the HTTP Request
 * `body`: string|byte() - Body of the HTTP Request, you will have to parse it in your handler to use it properly.
 * `headers`: map(string)string - HTTP Request Headers
```

Comment:
Should we use the canonical types for each language? I'm not sure

Here's what it would look like for python:
```suggestion
* `path`: str - Parameters defined in the path of the HTTP Request
* `queryStringParameters`: dict[str, str] - Query Strings parameters of the HTTP Request
* `body`: str - Body of the HTTP Request, you will have to parse it in your handler to use it properly.
* `headers`: dict[str, str] - HTTP Request Headers
```